eoscostarica/eoscostarica.io
# Apply new design services

### Apply new desing
### Refactor code

#120
#225

### Steps to test
1. remove curly braces
2. remove semicolon
3. remove spaces on clsx
4. check break lines on H1 TAGS
1. apply new design
1. add process image
2. add map image
3. delete other map image
